crates/driver/src/domain/liquidity/zeroex.rs

```diff
@@ -30,6 +31,12 @@ pub struct Order {
     pub signature: ZeroExSignature,
 }
 
+#[derive(Clone, Debug)]
+pub struct AdjustedAmounts {
```

Comment:
I think calling this `FillableAmounts` an a small doc comment would help understanding why this is needed in the first place.

Reply:
Renamed and added docs.